main_Poisson.C

```diff
@@ -19,26 +19,40 @@
 #include "Profiler.h"
 
 
+/*!
+  \brief Struct with additional simulation options from command-line.
+*/
+
+struct PoissonArgs
```

Comment:
why not inheriting SIMargsBase to avoid the two argument containers?

Reply:
In did not see any gain from it in the first glance since it was not passed to the runSimulator method. But if it helps readability, why not.